cfgov/v1/tests/test_wagtail_hooks.py

```diff
@@ -247,3 +250,100 @@ def test_tag_param_returns_resources_tagged_that_way(self):
             response.context['object_list'][1].title,
             'Test resource Banana'
         )
+
+
+class TestWhitelistOverride(TestCase):
+    # Borrowed from https://github.com/wagtail/wagtail/blob/v1.13.4/wagtail
+    # /wagtailcore/tests/test_dbwhitelister.py
+    def assertHtmlEqual(self, str1, str2):
```

Comment:
Can you use the built-in Django [`assertHTMLEqual`](https://docs.djangoproject.com/en/1.11/topics/testing/tools/#django.test.SimpleTestCase.assertHTMLEqual) here? (Helpfully documented in [our docs](https://github.com/cfpb/development/blob/master/guides/unittesting-django-wagtail.md#choosing-a-testcase-base-class)!)

Reply:
Perhaps I can!

Reply:
Done in 4f4317f!